akka-docs/rst/java/code/docs/pattern/SupervisedAsk.java

```diff
@@ -61,13 +58,10 @@ public Receive createReceive() {
 
     @Override
     public SupervisorStrategy supervisorStrategy() {
-      return new OneForOneStrategy(0, Duration.Zero(),
-          new Function<Throwable, Directive>() {
-            public Directive apply(Throwable cause) {
-              caller.tell(new Status.Failure(cause), self());
-              return SupervisorStrategy.stop();
-            }
-          });
+      return new OneForOneStrategy(0, Duration.Zero(), cause -> {
+          caller.tell(new Status.Failure(cause), self());
+          return SupervisorStrategy.stop();
+        });

```

Comment:
Of course not part of this PR, but this reminds me that we should get #20891 going, to have consistent formatting in java files.